cupy/math/window.py

```diff
@@ -87,3 +90,50 @@ def hanning(M):
         return basic.ones(1, float)
     n = ranges.arange(0, M)
     return 0.5 - 0.5 * trigonometric.cos(2.0 * numpy.pi * n / (M - 1))
+
+
+@util.memoize()
+def _kaiser_kernel():
+    return ElementwiseKernel("float32 beta, float32 alpha",
```

Comment:
have this in a global variable
```
_kaiser_kernel = ElementwiseKernel(...)
```

Reply:
sorry I misunderstood what you said about the global variable.